index.js

```diff
@@ -281,13 +282,16 @@ function processStandardCollection(auth) {
     core.setFailed("GURU_CONVERT_MARKDOWN is only supported for EXTERNAL collections: https://developer.getguru.com/docs/guru-sync-manual-api");
     return;
   } else {
+    const cardFooter = process.env.GURU_CARD_FOOTER?`\n---\n${process.env.GURU_CARD_FOOTER}\n`:'';
     let cardConfigs = yaml.parse(fs.readFileSync(process.env.GURU_CARD_YAML, 'utf8'));
     for (let cardFilename in cardConfigs) try {
+      const cardContent = fs.readFileSync(cardFilename, 'utf8');
```

Comment:
looks like you're creating a const and then immediately trying to update it. should probably use `let` instead

Reply:
Gah, missed that. Thanks! Should be good to test now?

Reply:
i think you updated the wrong one. `cardContent` needs to be changed to `let`

I'm still seeing the error 

I'm also not familiar with every branch in the logic so idk how relevant these are, but at a glance there are a few more consts you're trying to overwrite at some point...

* [tmpfileBase](https://github.com/peckjon/github-to-guru/pull/26/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R109) being changed [here](https://github.com/peckjon/github-to-guru/pull/26/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R111)
* [cardConfig](https://github.com/peckjon/github-to-guru/pull/26/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R126) being changed [here](https://github.com/peckjon/github-to-guru/pull/26/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R128)
* [htmlLinkMatches](https://github.com/peckjon/github-to-guru/pull/26/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R157) being changed [here](https://github.com/peckjon/github-to-guru/pull/26/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R161)
* [mdLinkMatches](https://github.com/peckjon/github-to-guru/pull/26/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R158) being changed [here](https://github.com/peckjon/github-to-guru/pull/26/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R162)